ST_grid.c

```diff
@@ -2632,7 +2632,7 @@ static void _do_seed_dispersal(void)
 				sgerm = (grid_SD[s][i].seeds_present
 						|| grid_SD[s][i].seeds_received) && germ; //refers to whether the species has seeds available from the previous year and conditions are correct for germination this year
 				grid_Species[s][i].allow_growth = FALSE;
-				biomass = grid_Species[s][i].relsize
```

Comment:
This line no longer references grid_Species because we are going to start using shallow copies which means s will refer to the correct species in the correct grid cell.

Reply:
This is just a comment, see issue #262 for information on how we plan on implementing gridded code changes.